core/trino-main/src/main/java/io/trino/memory/MemoryInfo.java

```diff
@@ -15,31 +15,23 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.collect.ImmutableMap;
-import io.airlift.units.DataSize;
-import io.trino.spi.memory.MemoryPoolId;
 import io.trino.spi.memory.MemoryPoolInfo;
 
-import java.util.Map;
-
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static java.util.Objects.requireNonNull;
 
 public class MemoryInfo
 {
     private final int availableProcessors;
-    private final DataSize totalNodeMemory;
```

Comment:
why is this gone? related?

Reply:
It was sum of `MemoryPool.maxBytes` over all pools. No longer useful if we have just one pool